src/style.css

```diff
@@ -0,0 +1,106 @@
+* {
+  box-sizing: border-box;
+  /* margin: 0; */
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  width: 200px;
+  padding: 30px 14px 14px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image-wrapper {
+  display: flex;
+  padding: 0 5px;
+  justify-content: center;
+  align-items: center;
+  /* margin-bottom: 45px; */
+}
+
+.card__image {
+  width: 100%;
+  /* width: 160px; */
+  height: 134px;
+}
+
+.card__title {
+  font-size: 12px;
+  line-height: 15px;
+  font-weight: 500;
+  margin-top: 45px;
+  margin-bottom: 0;
+}
+
+.card__text {
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.card__box {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.ratings {
+  margin-top: 14px;
+}
+
+.price {
+  margin-top: 20px;
+  
+}
+
+.ratings__stars {
+  display: flex;
+}
+
+.ratings__star {
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.ratings__stars .ratings__star:nth-child(-n+4) {
```

Comment:
Try to avoid increasing the specificity of the selector. One class is enough to identify the elements on the page. No need to write something like .block.colored-block. Use .block and .block--colored instead, and apply each of them to your element